.gitignore

```diff
@@ -0,0 +1,12 @@
+_skbuild
+_deps
+vtk-wheel-sdk-build
+vtk-wheel-sdk-subbuild
+vtkexternalmodule-build
+vtkexternalmodule-subbuild
+wheelhouse
+.venv
+__pycache__
+*.egg-info
+pyi
```

Comment:
If these a generated by VTK, I suggest to move it under the `VTKU3DExporter` section